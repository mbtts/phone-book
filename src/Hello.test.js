import Hello from "./Hello";
import React from "react";
import { shallow } from "enzyme";

describe("Hello", () => {
  it("should render initial state", () => {
    const wrapper = shallow(<Hello />);
    expect(wrapper).toMatchSnapshot();
  });
});
