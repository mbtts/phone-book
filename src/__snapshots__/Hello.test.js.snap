// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render initial state 1`] = `
<div>
  Hello World
</div>
`;
